.github/workflows/release.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
 
   release:
     needs: check
-    uses: canonical/bootstack-actions/.github/workflows/charm-release.yaml@v2
+    uses: canonical/bootstack-actions/.github/workflows/charm-release.yaml@v3
```

Comment:
We're moving away from the bootstack-actions.  Can we switch to using the upload-charm workflow directly?  See https://github.com/canonical/charm-local-users/blob/main/.github/workflows/release.yaml for a recent example of what we're moving to.

Reply:
@samuelallan72 Added the same workflow, just removed `runs-on`.